File: library/src/main/java/cn/cqray/android/dialog/amin/FadeIn.java
package cn.cqray.android.dialog.amin;

import androidx.annotation.NonNull;

import cn.cqray.android.anim.AnimatorBuilder;

/**
 * 淡入
 * @author Cqray
 */
public class FadeIn extends DialogAnimator {

    @Override
    public void onHandle(@NonNull AnimatorBuilder builder) {
        builder.fadeIn();
    }
}

File: library/src/main/java/cn/cqray/android/dialog/amin/FadeIn.kt
package cn.cqray.android.dialog.amin

import cn.cqray.android.anim.ViewAnimator

/**
 * 淡入
 * @author Cqray
 */
class FadeIn : GetAnimator() {
    override fun onAnimatorPrepared(animator: ViewAnimator) {
        animator.fadeIn()
    }

}